src/pages/Settings.tsx

```diff
@@ -1,14 +1,16 @@
 import React, {useEffect} from "react";
 import {useForm} from "antd/es/form/util";
-import {Button, Card, Form, Input, notification} from "antd";
+import {Button, Card, Form, Input, notification, Select} from "antd";
 import {formatMessage, FormattedMessage} from "umi-plugin-react/locale";
 import {router} from "umi";
 import {PageHeaderWrapper} from "@ant-design/pro-layout";
-import {getAdminAPI, getAdminAPIKey} from "@/utils/utils";
+import {getAdminAPIConfig} from "@/utils/utils";
```

Comment:
It's not a good idea to name as `/utils/utils`, please use something like `/utils/setting` or `/utils/config` instead.

Reply:
or use `/utils` file directly.

Reply:
use `/utils/setting`